common/utils/remote.go

```diff
@@ -0,0 +1,29 @@
+package utils
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
What about calling this file `ssh` ?